docs/admin/logging/logentries.md

```diff
@@ -0,0 +1,42 @@
+<!--[metadata]>
++++
+aliases = ["/engine/reference/logging/logentries/"]
+title = "Logentries logging driver"
+description = "Describes how to use the Logentries logging driver."
+keywords = ["logentries, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging"
+weight = 2
++++
+<![end-metadata]-->
+
+# Logentries logging driver
```

Comment:
This document was moved to the central docs repository. Given that will be part of 1.13, and should not be published yet, can you remove this file here, and open a PR against the `vnext-engine` branch in that repository?

https://github.com/docker/docker.github.io/tree/vnext-engine/engine/admin/logging